### ISSUE TITLE
Stories: Media Picker should open by default when opening Story editor

### ISSUE THREAD

**Issue by thehenrybyrd** [wordpress-mobile/WordPress-Android#14617]
### Expected behavior

* Story interface should open Media Picker by default

### Actual behavior

* Story interface opens camera by default

### Steps to reproduce the behavior

* Create a new Story Post
* Notice the camera preview is shown by default

##### Tested on Samsung Galaxy S21, Android 11, WPAndroid 17.3-rc-1
Duplicating issue from iOS: https://github.com/wordpress-mobile/WordPress-iOS/issues/16446

**Comment by aforcier** [comment]
@thehenrybyrd I wasn't able to reproduce this on WPAndroid 17.3-rc-1 - the media picker is shown by default when creating a story from the create menu.

We _do_ show the camera first on that build when creating a new blog post > adding a new story block > adding media, perhaps that's the path you tried? That was just fixed to show the media picker first, it's in develop so would be in 17.4 (https://github.com/wordpress-mobile/WordPress-Android/pull/14588).

**Comment by thehenrybyrd** [comment]
@aforcier you're completely right, this issue is incorrect. I hastily copied https://github.com/wordpress-mobile/WordPress-iOS/issues/16446 over from iOS when a beta tester found a related issue on both iOS and Android. Turns out what the tester found is a separate issue, which I've now filed at https://github.com/wordpress-mobile/WordPress-Android/issues/14646.
More context at p5T066-2fh-p2#comment-8242
Closing this issue.

Thanks for the comment, it prompted me to take a closer look!
